start-workflow-from-archive/start-multiple-workflows.sh

```diff
@@ -0,0 +1,110 @@
+#!/bin/sh
```

Comment:
You use a lot of bashism leading this script to probably fail on systems where Bash is not the default shell (e.g. Debian or Ubuntu). For details, take a look at the output of `shellcheck`:

```

In start-multiple-workflows.sh line 30:
POSITIONAL=()
           ^-- SC2039: In POSIX sh, arrays are undefined.


In start-multiple-workflows.sh line 31:
while [[ $# -gt 0 ]]
      ^-- SC2039: In POSIX sh, [[ ]] is undefined.


In start-multiple-workflows.sh line 46:
    PROPERTIES+=(" $2")
    ^-- SC2039: In POSIX sh, += is undefined.
                ^-- SC2039: In POSIX sh, arrays are undefined.


In start-multiple-workflows.sh line 66:
    POSITIONAL+=("$1") # save it in an array for later
    ^-- SC2039: In POSIX sh, += is undefined.
                ^-- SC2039: In POSIX sh, arrays are undefined.


In start-multiple-workflows.sh line 71:
set -- "${POSITIONAL[@]}" # restore positional parameters
        ^-- SC2039: In POSIX sh, array references are undefined.


In start-multiple-workflows.sh line 74:
if [ "x" == "x$WORKFLOW" ]; then
         ^-- SC2039: In POSIX sh, == in place of = is undefined.


In start-multiple-workflows.sh line 78:
    ARGS+=" -w $WORKFLOW"
    ^-- SC2039: In POSIX sh, += is undefined.


In start-multiple-workflows.sh line 81:
if [ "x" != "x$PROPERTIES" ]; then
              ^-- SC2128: Expanding an array without an index only gives the first element.


In start-multiple-workflows.sh line 82:
    for p in "${PROPERTIES[@]}"; do
              ^-- SC2039: In POSIX sh, array references are undefined.


In start-multiple-workflows.sh line 83:
        ARGS+=" -W $p"
        ^-- SC2039: In POSIX sh, += is undefined.


In start-multiple-workflows.sh line 88:
    ARGS+=" -o $OC_SERVER_URL"
    ^-- SC2039: In POSIX sh, += is undefined.


In start-multiple-workflows.sh line 92:
    ARGS+=" -u $OC_USER"
    ^-- SC2039: In POSIX sh, += is undefined.


In start-multiple-workflows.sh line 96:
    ARGS+=" -p $OC_PASSWORD"
    ^-- SC2039: In POSIX sh, += is undefined.


In start-multiple-workflows.sh line 102:
    [ "x" == "x$mp" ]  && exit 0
          ^-- SC2039: In POSIX sh, == in place of = is undefined.


In start-multiple-workflows.sh line 104:
    python3 StartWorkflow.py -m $mp $ARGS || /bin/true
                                ^-- SC2086: Double quote to prevent globbing and word splitting.
                                    ^-- SC2086: Double quote to prevent globbing and word splitting.
```

If you want to use bash, explicitely require bash.